src/main/kotlin/snyk/container/ui/ContainerIssueDetailPanel.kt

```diff
@@ -0,0 +1,156 @@
+package snyk.container.ui
+
+import com.intellij.uiDesigner.core.GridConstraints
+import com.intellij.uiDesigner.core.GridLayoutManager
+import com.intellij.uiDesigner.core.Spacer
+import icons.SnykIcons
+import io.snyk.plugin.Severity
+import io.snyk.plugin.ui.baseGridConstraints
+import io.snyk.plugin.ui.baseGridConstraintsAnchorWest
+import io.snyk.plugin.ui.descriptionHeaderPanel
+import io.snyk.plugin.ui.getReadOnlyClickableHtmlJEditorPane
+import io.snyk.plugin.ui.panelGridConstraints
+import org.commonmark.parser.Parser
+import org.commonmark.renderer.html.HtmlRenderer
+import snyk.container.ContainerIssue
+import java.awt.Font
+import java.awt.Insets
+import javax.swing.JEditorPane
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+class ContainerIssueDetailPanel(
+    private val issue: ContainerIssue
+) : JPanel() {
+
+    init {
+        this.layout = GridLayoutManager(10, 1, Insets(20, 10, 20, 20), -1, 10)
```

Comment:
the numbers should be constants describing what it does

Reply:
see reply above